module/Activity/view/email/activity.phtml

```diff
@@ -0,0 +1,37 @@
+<?php $lang = 'en'; ?>
+<?= $this->translate('Hello,')?><br /><br />
```

Comment:
I think it is fine to just have the email in one language. Which of the two shouldn't really matter, however the person reading it most likely will be dutch.